index.bs

```diff
@@ -1290,9 +1309,7 @@ An {{XRPose}} describes a position and orientation in space relative to an {{XRS
 
 The <dfn attribute for="XRPose">transform</dfn> attribute describes the position and orientation relative to the base {{XRSpace}}.
 
-<section class="unstable">
 The <dfn attribute for="XRPose">emulatedPosition</dfn> attribute MUST be <code>false</code> if the {{XRPose/transform}}.{{XRRigidTransform/position}} values are based on sensor readings, or <code>true</code> if the position values are software estimations, such as those provided by a neck or arm model.
```

Comment:
nit: "based exclusively on sensor readings"?

Reply:
Yea, this is a tricky thing to phrase to both meet developer expectations and not presume too much about how an implementation will work, since tracking systems today generally involve both "sensor readings" _and_ IMU inference, even during nominal 6DoF tracking.

Thanks to @toji, we now have far more precise spec language about exactly when `emulatedPosition` MUST be true or false in the switch at [line 852](https://github.com/immersive-web/webxr/pull/559/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR852).

Perhaps we just let that language control instead, and remove the normative clauses here:
> The `emulatedPosition` attribute is `false` when the `XRPose/transform` represents an actively tracked [=6DoF=] pose based on sensor readings, or `true` if its `XRRigidTransform/position` value includes a computed offset, such as that provided by a neck or arm model.

Reply:
I'd gone with the less normative text above.